code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/identity/TokenResourceManager.java

```diff
@@ -61,6 +65,7 @@ private Token createToken(ApiRequest request) {
             throw new ClientVisibleException(ResponseCodes.SERVICE_UNAVAILABLE,
                     "codeTypeInvalid", "Code type provided invalid.", null);
         }
+        token.setJwt(authTokenDao.createToken(token.getJwt(), token.getAuthProvider()).getKey());
```

Comment:
I feel we should rename to setJwtKey() since that's really the value we're passing in rather than the jwt itself